src/example_ws_usage.ts

```diff
@@ -3,12 +3,15 @@ import { PythConnection } from './PythConnection'
 import { getPythClusterApiUrl, getPythProgramKeyForCluster, PythCluster } from './cluster'
 import { PriceStatus } from '.'
 
-const SOLANA_CLUSTER_NAME: PythCluster = 'mainnet-beta'
+const SOLANA_CLUSTER_NAME: PythCluster = 'devnet'
 const connection = new Connection(getPythClusterApiUrl(SOLANA_CLUSTER_NAME))
 const pythPublicKey = getPythProgramKeyForCluster(SOLANA_CLUSTER_NAME)
 
 const pythConnection = new PythConnection(connection, pythPublicKey)
-pythConnection.onPriceChange((product, price) => {
+pythConnection.onPriceChangeVerbose((productAccount, priceAccount) => {
+  // The arguments to the callback include solana account information / the update slot if you need it.
+  const product = productAccount.accountInfo.data.product;
+  const price = priceAccount.accountInfo.data;
```

Comment:
do we also wanna keep the `onPriceChange` example or do we want to encourage new users to use `onPriceChangeVerbose`?

Reply:
I think most people will prefer the verbose function (because you almost always need the account keys), hence why i used that in the example.